### DISCUSSION TITLE
See job queue

### DISCUSSION THREAD

**Discussion by Miker91** [actions/actions-runner-controller#3038] [Questions]
Hi,

I'm wondering if there is possibility to see the queue of jobs waiting to be picked up by ARC? 
I know Prometheus monitoring is possible, but I'm thinking about list of jobs including job name and repo name.

Thanks,
Mateusz

**Comment by rekha-prakash-maersk** [comment]
Hi @Miker91 , I am also looking for same metrics. you mentioned that prometheus metrics is possible. how can I use that one to start with?

**Reply by Miker91** [reply]
Hi @rekha-prakash-maersk. 
Prometheus metrics are exposed by default when you install ARC from helm chart. 
You can also modify values to amend Prometheus port, proxy etc. 
Please find more info here: https://github.com/actions/actions-runner-controller/blob/master/docs/monitoring-and-troubleshooting.md
